src/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { URL } from 'url';
+
+/**
+ * Checks if the argument is URL or not
+ * @param {String} path Path to document
+ * @returns {Boolean} Whether the given path is URL or not
+ */
+export function isUrl(path: string): boolean {
+  if (path.startsWith('http')) {
+    new URL(path);
+    return true;
+  }
+  return false;
+}
+
+/**
+ * Synchronously reads the entire content of passed document file
+ * @param {String} path Path to document
+ * @returns {String} The entire content of a file
+ */
+export function readDocument(path: string): string {
+  const resolvedPath = resolve(path);
+  return readFileSync(resolvedPath, 'utf-8');
```

Comment:
@aayushmau5, Is it necessary to read the file synchronously. What if we read it in an async fashion? Doing so will increase the performance of our code. Please share your views.

Reply:
Yeah, we can do it asynchronously. The reason I did is synchronously to just "safeguard" that the file has been read before continuing the next step (which, I think now, does not even make sense :laughing: ).

I'm going to push changes with `async`. :+1: